nautobot/utilities/utils.py

```diff
@@ -915,17 +915,9 @@ def get_filterset_parameter_form_field(model, parameter, filterset=None):
                 choices_as_strings=True, queryset=queryset_map[plural_name]().as_queryset()
             )
     elif isinstance(field, (filters.MultipleChoiceFilter, filters.ChoiceFilter)) and "choices" in field.extra:
```

Comment:
This may also be a bit incorrect - I don't think we should be presenting a multiple-choice formfield/widget for a `filters.ChoiceFilter` - but it's outside the narrow scope of fixing this bug.